### PR TITLE
Add links to platform-specific guides to setting up a development environment

### PR DESCRIPTION
**Pull request by ghost**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Description

~~Add the command to remove all files named `.DS_Store` from the repo's directory tree. These Mac cache files, if present in `/seed` or its subdirectories, prevent `node seed` from successfully saving challenges to MongoDB.~~

Modify the Contributor's Guide and add links to platform-specific guides to setting up a development environment.

### CODE REVIEW

**Comment by raisedadead** [comment]
@leonfeng thanks for the pull request.

@freeCodeCamp/moderators : 
I think we should add this to an article on our wiki. And linking it here in the guide. I would love to see an article with common pain points of the setup with various OS-es, as a FAQ, rather than cluttering up the contributing guide.

We already feel that the setup throws off so many new contributors.

Thoughts?

**Comment by erictleung** [comment]
@raisedadead I agree. The contributing document can be unwieldy to newcomers. We do have a Wiki on [how to close and setup the freeCodeCamp website on a Windows machine](https://forum.freecodecamp.com/t/19366), so I don't see any issues with having article(s) for OS specific instructions.

**Comment by raisedadead** [comment]
@leonfeng would you be interested in creating an article as suggested and update this pull request to link it up on the guide?

**Comment by BerkeleyTrue** [comment]
Since this a common problem (I've seen @QuincyLarson run into this several times) I'm working on updating the seed script to ignore dotfiles.

**Comment by leonfeng** [comment]
@raisedadead Sure thing. Working on it right now.

**Comment by BerkeleyTrue** [comment]
Dotfiles will no longer break seeding task. Please remove that line

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -216,6 +220,10 @@ congratulations – you're all set. Otherwise, let us know by asking in the [Con
  # Re-run bower install
  bower install
 
+ # .DS_Store can prevent `node seed` from saving challenges to the database
```
Comment (raisedadead) [inline CONTRIBUTING.md]: Remove lines 223 to 226

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @leonfeng

**Comment by QuincyLarson** [comment]
@leonfeng Thanks! This is a welcome improvement!

**Pull request merged**